dependency_support/pip_requirements.txt

```diff
@@ -8,6 +8,8 @@ termcolor==1.1.0
 psutil==5.7.0
 portpicker==1.3.1
 pyyaml==6.0.1
+git+https://github.com/cocotb/cocotb.git@a853db95b0019db6796a6803aa94304bde743e4e
```

Comment:
can we ping the cocotb folks to do a release?

Reply:
any news about a new cocotb release?

Reply:
The official release is in progress. I asked about the date in https://github.com/cocotb/cocotb/issues/3369#issuecomment-1729325948

Reply:
It looks like they are making steady progress towards release: https://github.com/cocotb/cocotb/issues/3369#issuecomment-1743806029 & https://github.com/cocotb/cocotb/pull/3429